### PR TITLE
Re-enable Canonical Ads with e2e fixes

### LINKED ISSUE #6182: Add GNL Google Ad to Canonical Ad Container
**Describe the solution you'd like**
Based on an investigation into implementing AMP Ads in Canonical pages the conclusion was reached that ads can be served using GTP (Google Publisher Tags) https://developers.google.com/doubleclick-gpt/guides/get-started

This will still serve AMP Ads but without the need for AMPHTML which caused event issues in Canonical pages, and likely is not supposed to be used outside AMP pages.

Here is a working example on the front page. 
https://github.com/bbc/simorgh/pull/6119/files

Further considerations needed:
- Props used in the AMP implementation should be ported over where possible. In this example only the prop `'data-slot': '/4817/bbccom.test.site.amp.news',` was used.
- Ad should be 970x250px and centred below the nav bar. **Further UX review will be required and mobile views should be considered**
- Should be enabled on the same pages as the AMP Ads, and implemented into toggles in the same way. 

**Describe alternatives you've considered**

These were also investigated but concluded GPT was the best solution.
- Using AMPHTML Ads directly on page
- Using AMPHTML Ads via iFrame

**Testing notes**
Consider creating an integration test in this final stage so ensure the ad is showing. Not yet sure how we test the user is in or out of an advertising region. No need to test contents of the ad.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by mmcculloch**
Resolves #6182 

**Overall change:**
Re-enable Canonical Ads with a few fixes

**Code changes:**

- GNL scripts now pushed to header with react-helmet, this prevents a race-condition error that would sometimes occur
- Disabled Ads on `zhongwen` sites due to a e2e error. Will raise another issue for this.
- Turned on ads-debug in order to disable geoIp check
- Check window.dotcom exists (taken from examples)

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [x] This PR requires manual testing

### CODE REVIEW

File: src/app/containers/Ad/Canonical/AdSlot.jsx [hunk 1]
```diff
@@ -1,12 +1,25 @@
...
+
+    return () => {
+      if (window.dotcom) {
+        window.dotcom.cmd.push(() => {
+          window.dotcom.ads.destroySlot(uniqueId);
+        });
+      }
+    };
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [location]);
```
Comment (thekp) [inline src/app/containers/Ad/Canonical/AdSlot.jsx]: Just out of curiosity, why did you remove `uniqueId` from here ^

File: src/app/containers/Ad/index.jsx [hunk 2]
```diff
@@ -16,9 +18,15 @@ const AdContainer = () => {
...
 
   if (isAmp) {
-    return <Amp />;
+    return <AmpAd />;
   }
-  return null;
+
+  return (
+    <>
+      <CanonicalAdBootstrap />
```
Comment (tochwill) [inline src/app/containers/Ad/index.jsx:26]: Howcome the the ad bootstrap isn't just in the CanonicalAd component?

**Review by tochwill — COMMENTED** [review]
Looks good! Just have one question

File: src/app/containers/Ad/index.jsx (hunk 2 above)
Comment (mmcculloch) [reply src/app/containers/Ad/index.jsx:26]: I think either way we need to put some consideration into moving this out of the component completely, if there are multiple ads we don't want it running multiple times.

I think this makes it a little clearer that it isn't supposed to be embedded in the Ad. If that makes any sense?

**Review by thekp — APPROVED** [review]
Looks good to me.

Just to note, our toggles API service is currently broken on TEST env, so I rendered the Ad by removing the enabled condition in the container.


image

**Review by tochwill — APPROVED** [review]
